client/js/button-view.js

```diff
@@ -0,0 +1,31 @@
+class ButtonView {
```

Comment:
Styling: set your IDE to use 2 spaces for tabs instead of 4. Ask me if you can't find where to do that in your settings.